### ISSUE TITLE
When testing the pretrained model with NIV dataset with pre extracted features, I get the following error. How do we solve this?

### ISSUE THREAD

**Issue by Ravindu-Yasas-Nagasinghe** [JoeHEZHAO/procedure-planing#1]
/apps/local/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1245: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
For epoch 0 using viterbi-algorithm, Best Success Rate 0.24024024024024024, meanIOU 0.7115415744786618 and meanACC 0.4719712063502642
/apps/local/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py:1245: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
For epoch 0 using argmax, Best Success Rate 0.15315315315315314, meanIOU 0.7195477964415962 and meanACC 0.4384098323270614

**Comment by JoeHEZHAO** [comment]
Hi, don't worry, they are just warnings from the used packages.
